apps/test/unit/StudioAppTest.js

```diff
@@ -0,0 +1,71 @@
+import sinon from 'sinon';
+import {expect} from '../util/configuredChai';
+import {singleton as studioApp, stubStudioApp, restoreStudioApp} from '@cdo/apps/StudioApp';
+import {throwOnConsoleErrors, throwOnConsoleWarnings} from '../util/testUtils';
+import {assets as assetsApi} from '@cdo/apps/clientApi';
+import {listStore} from '@cdo/apps/code-studio/assets';
+
+describe('StudioApp.singleton', () => {
+  throwOnConsoleErrors();
+  throwOnConsoleWarnings();
+
+  beforeEach(stubStudioApp);
+  afterEach(restoreStudioApp);
+
+  let containerDiv, codeWorkspaceDiv;
+  beforeEach(() => {
+    codeWorkspaceDiv = document.createElement('div');
+    codeWorkspaceDiv.id = 'codeWorkspace';
```

Comment:
Is this still necessary after you last commit? (That appears to make us handle the case where we dont have a codeWorkspace)